onnxruntime/test/providers/qnn/qnn_test_utils.h

```diff
@@ -239,6 +249,19 @@ void InferenceModel(const std::string& model_data, const char* log_id,
  */
 void TryEnableQNNSaver(ProviderOptions& qnn_options);
 
+struct QDQTolerance {
+  // When comparing output activations between QNN EP and CPU EP (both running the QDQ model),
+  // this value defines the maximum tolerable difference as a percentage of the output range.
+  // Ex: (qdq@QNN_EP - qdq@CPU_EP) / (rmax_output - rmin_output) <= DEFAULT_QDQ_TOLERANCE.
+  static constexpr float DEFAULT_QDQ_TOLERANCE = 0.004f;  // 0.4% is equivalent to 1 int8 quantization unit
+                                                          // or 262 int16 quantization units.
```

Comment:
Note: previous used 1 quantization unit + epsilon as the allowed tolerance between qdq model results. Now, reframing this tolerance as a percentage of the output range. 1 quantization unit is 1/255 = 0.4% for 8bit.